plugin/trino-hive/src/main/java/io/trino/plugin/hive/TrinoViewHiveMetastore.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
`Refactor Iceberg views to use helper class` -> `Extract Trino View functionlity`